### DISCUSSION TITLE
Can I open File explorer to choose files?

### DISCUSSION THREAD

**Discussion by kznr02** [hecrj/iced#1650] [Q&A] [answered]
I can't find any API of iced to reach this goal, I'm using Windows, I want to code a tool which has a function to choose a local file in my computer, just like I click a button then a dialog pops up to show me the file explorer

**Comment by 13r0ck** [comment, marked as answer]
https://crates.io/crates/rfd

Use the rfd async api, and use a command to await `AsyncFileDialog::new().pick_file()` and then return whatever you want from the file to your application.

**Reply by kznr02** [reply]
Thanks 👍
